src/main/java/com/julienvey/trello/impl/TrelloImpl.java

```diff
@@ -510,6 +438,18 @@ public void addLabelsToCard(String idCard, String[] labels) {
         }
     }
 
+    @Override
+    public CustomFieldItem updateCustomFieldItem(String idCard, CustomFieldItem customFieldItem) {
+        Objects.requireNonNull(idCard);
+        Objects.requireNonNull(customFieldItem);
+
+        customFieldItem = put(createUrlWithNoArgs(UPDATE_CUSTOM_FIELD_ITEM), customFieldItem, CustomFieldItem.class, idCard,
```

Comment:
let's remove reassigning param value to anything.
it is better to create a new instance